service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/EhrExtractStatusService.java

```diff
@@ -401,12 +401,15 @@ public void updateEhrExtractStatusAck(String conversationId, EhrReceivedAcknowle
         if (hasAcknowledgementExceededEightDays(conversationId, ack.getReceived())
             && hasEhrStatusReceivedAckWithUnexpectedConditionErrors(conversationId)) {
 
-            logger().warn("Received an ACK message with a conversation_id: {}, but it will be ignored", conversationId);
+            logger().warn("Received an ACK message with conversation_id: {}, "
+                          + "but it is being ignored because the EhrExtract has already been marked as failed "
+                          + "from not receiving an acknowledgement from the requester in time.",
+                          conversationId);
             return;
         }
 
         if (hasFinalAckBeenReceived(conversationId)) {
-            logger().warn("Received an ACK message with a conversation_id=" + conversationId + " that is a duplicate");
+            logger().warn("Received an ACK message with a conversation_id: {} that is a duplicate", conversationId);
```

Comment:
May be out of scope for this ticket, although I am still a little confused why we need:

```java
protected Logger logger() {
    return LOGGER;
}
```
Surely adding `@Slf4j` at class-level, we can then remove this method completely - allowing up to list call `log.warn()`?

Reply:
I just find it as an easy way of testing logger but at the same time encapsulating it which is also a valid technique as far as I know.

Reply:
I understand where you're coming from, although Lombok's documentation states:

Example:
```java
 @Slf4j
 public class LogExample { }
```

will generate:
```java
public class LogExample {
   private static final org.slf4j.Logger log = org.slf4j.LoggerFactory.getLogger(LogExample.class);
}
```